charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -233,6 +233,318 @@ containerLogs:
             log_stream_prefix   ${HOST_NAME}.
             auto_create_group   true
             extra_user_agent    container-insights
+      ADCISOextraFiles:
```

Comment:
Could we rename this section to follow Helm naming conventions - https://v2-14-0.helm.sh/docs/chart_best_practices/#naming-conventions